LRRServer/lawreadingrobot/models.py

```diff
@@ -0,0 +1,48 @@
+
+import uuid
+
+from sqlalchemy import JSON, Column, DateTime, Text, Uuid
+from sqlalchemy.sql import func
+
+from .database import Base
+
+
+class BaseMixin(object):
```

Comment:
Please name this something that describes it's contents, e.g. "TimestampMixin".

Reply:
Seconded.